.github/report-handling-manual.md

```diff
@@ -0,0 +1,92 @@
+
+The Code of Conduct Committee should aim to have a resolution agreed upon within two weeks. In the event that a resolution can’t be determined in that time, the Committee will respond to the reporter(s) with an update and projected timeline for resolution.
+
+
+## Resolutions
+
+The Committee must agree on a resolution by consensus. If the group cannot reach consensus and deadlocks for over a week, the group will turn the matter over to the NumFOCUS Code of Conduct Enforcement Team for resolution.
+
+Possible responses may include:
+
+* Taking no further action:
+  -   if we determine no violations have occurred;
+  -   if the matter has been resolved publicly while the Committee was considering responses.
+* Coordinating voluntary mediation: if all involved parties agree, the Committee may facilitate a mediation process as detailed above.
+* Remind publicly, and point out that some behavior/actions/language have been judged inappropriate and why in the current context, or can but hurtful to some people, requesting the community to self-adjust.
+* A private reprimand from the Committee to the individual(s) involved. In this case, the group chair will deliver that reprimand to the individual(s) over email, cc’ing the group.
+* A public reprimand. In this case, the Committee chair will deliver that reprimand in the same venue that the violation occurred, within the limits of practicality. E.g., the original mailing list for an email violation, but for a chat room discussion where the person/context may be gone, they can be reached by other means. The group may choose to publish this message elsewhere for documentation purposes.
+* A request for a public or private apology, assuming the reporter agrees to this idea: they may at their discretion refuse further contact with the violator. The chair will deliver this request. The Committee may, if it chooses, attach “strings” to this request: for example, the group may ask a violator to apologize in order to retain one’s membership on a mailing list.
+* A “mutually agreed upon hiatus” where the Committee asks the individual to temporarily refrain from community participation. If the individual chooses not to take a temporary break voluntarily, the Committee may issue a “mandatory cooling off period”.
+* A permanent or temporary ban from some or all Cython spaces (mailing lists, gitter.im, etc.). The group will maintain records of all such bans so that they may be reviewed in the future or otherwise maintained.
```

Comment:
We're not using gitter in any official way. (At least, I'm not.) There's an IRC channel, though. Not sure if that still counts as official "Cython space".

Reply:
Removed Gitter, added GitHub instead. IRC - no idea about whether it's "official". Depends on who are the admins I'd say, if that's you and it's a place people can talk to devs, then probably yes.

For NumPy there's a gitter, a subreddit, etc. - we list those, but explicitly say we don't monitor them: https://numpy.org/gethelp/.  And we keep it on a separate page of the website ("get help" rather than "community").